Modix.Services/PopularityContest/PopularityContestService.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Discord;
+using Discord.WebSocket;
+using Humanizer;
+using Modix.Services.AutoCodePaste;
+using Modix.Services.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Modix.Services.PopularityContest
+{
+    public class PopularityContestService
```

Comment:
You mind throwing up an `IPopularityContestService` interface as well, for consistency and future testings?